homeassistant/components/teslemetry/coordinator.py

```diff
@@ -48,7 +48,7 @@ def flatten(data: dict[str, Any], parent: str | None = None) -> dict[str, Any]:
 class TeslemetryVehicleDataCoordinator(DataUpdateCoordinator[dict[str, Any]]):
     """Class to manage fetching data from the Teslemetry API."""
 
-    name = "Teslemetry Vehicle"
+    updated_once: bool = False
```

Comment:
As you can have more than one of these coordinators (I think) just define the typing on class level and set it at `__init__` so this variable isn't shared between instances.